ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -99,6 +99,15 @@ then
 	#then
 	#fi
 
+	if ver hwcmp AEROFC_V1
+	then
+		if param compare SYS_AUTOSTART 0
+		then
+			param set SYS_AUTOCONFIG 1
+			param set SYS_AUTOSTART 4070
+		fi
+	fi
```

Comment:
I don't think this is correct. It only works with 4070 autostart, because none of the others have the required components. Otherwise user may load a configuration he thinks will work, but it won't.

The way it was before was similar to crazyflie, setting AUTOCNF if there was no SYS_AUTOSTART configuration set. It's the same here, but setting AUTOCNF flag instead of SYS_AUTOCONFIG param.